en/contributing/build.md

```diff
@@ -124,16 +152,6 @@ build/default/integration_tests_runner --gtest_filter="ExternalExampleHello"
 ```
 
 
-
-### Code Style
-
-All cpp and h files should be formatted according to the astyle settings defined by astylerc.
-To automatically fix the formatting, run this command:
-
-```
-make fix_style
-```
-
 ## Building in Docker
```

Comment:
If I build in docker using this approach, what can I do/not do? 
- So afaik this just builds the main C++ library - is there intention to add android support in here?
- Say I use this  in my Windows hosted VM and build an example app that does something. HOw can I connect to the autopilot connected to my computer.
- Same question as above if I am running it direct on my Linux computer.